MainApp/public/app/controllers/gamePageCtrl.js

```diff
@@ -1,21 +1,24 @@
-angular.module('gamePageControllers', ['roomServices', 'circleServices', 'refreshServices'])
-    .controller('gamePageCtrl', function ($scope, $http, $routeParams, $timeout, $location, $anchorScroll, $rootScope, $cookies, PatientService, Room, Agent, Circle, GameState, UserStats, Refresh) {
+angular.module('gamePageControllers', ['roomServices', 'circleServices', 'refreshServices', 'scrollingServices'])
+    .controller('gamePageCtrl', function ($scope, $http, $routeParams, $timeout, $location, $anchorScroll, $rootScope,
+                            $cookies, PatientService, Room, Agent, Circle, GameState, UserStats, Refresh, Scrolling) {
         let app = this;
         let blinkTimer;
         let blinkTimer2;
         Refresh.checkRefresh();
         app.username=$rootScope.username;
         (function startButton() {
-            alert("The goal is to save as many patients as possible");
+            alert("The goal is to save as many patients as possible. Please switch to landscape mode if using a " +
```

Comment:
I can detect screen width and do that. However, even for tablets the experience is much better on landscape. And some tablets like the IPad pro are almost as big as some laptop screens. Plus some people orient their monitors vertically - where again the experience is not as good as in landscape. So not exactly sure what screen width I should use.